associations.rb

```diff
@@ -13,26 +13,27 @@ def migrate
 
 def generate_migrations
   ActiveRecord::Migration.create_table :hotels do |t|
-    #insert our columns here
-
+    t.string :name
+    t.integer :room_count
     t.timestamps null: false
   end
 
   ActiveRecord::Migration.create_table :rooms do |t|
-    #insert our columns here
-
+    t.float :rate
+    t.string :location
+    t.integer :hotel_id
     t.timestamps null: false
   end
 
   ActiveRecord::Migration.create_table :bookings do |t|
-    #insert our columns here
-
+    t.integer :room_id
+    t.integer :user_id
```

Comment:
For the fields that are foreign keys to other tables, you probably want to make sure to get the foreign key constraint  on there (either with the `belongs_to` method or with `add_reference`.  It's also a good habit to think about whether any of these columns should be non-nullable.